app/Http/Controllers/Admin/MonsterCrudController.php

```diff
@@ -1217,6 +1218,14 @@ public static function getFieldsArrayForUploadsTab()
             'tab' => 'Uploads',
         ];
 
+        $fields[] = [ // dropzone
+            'label' => 'Dropzone'.backpack_pro_badge(),
+            'name'  => 'dropzone',
+            'type'  => 'dropzone',
+            'hint'  => 'Some info',
+            'tab'   => 'Uploads',
+        ];
+
```

Comment:
I don't see any events here 👀 Right now we're not moving the file from the temporary directory to the final directory, correct?

Reply:
Example here - https://backpackforlaravel.com/docs/5.x/release-notes#define-model-events-directly-on-fields

<img width="800" alt="CleanShot 2023-02-07 at 20 14 15@2x" src="https://user-images.githubusercontent.com/1032474/217254655-1422ae56-deb9-46cf-be65-f65fde7bf498.png">

Reply:
Example event:
```php
        $fields[] = [ // dropzone
            'label' => 'Dropzone'.backpack_pro_badge(),
            'name'  => 'dropzone',
            'type'  => 'dropzone',
            'hint'  => 'Some info',
            'tab'   => 'Uploads',
            'events' => [
               'saving' => (function($entry) {
                    // Step 1. Get the temporary file path.
                    
                    // Step 2. Move that temp file to the final directory & rename it.
                    
                    // Step 3. Save the final path on the entry attribute.
                    $entry->image = $final_path;
                })
            ]
        ];
```